inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/HtmlAnnotationEditor.java

```diff
@@ -173,17 +173,6 @@ private void handleError(String aMessage, Throwable aCause)
         return;
     }
 
-    private String toJson(Object result)
```

Comment:
Having methods - even if they are used only once - helps making other methods shorter and therefore more readable. From my perspective, this is one of the cases where inlining the method makes the target method unnecessary long and less readable. The `toJson` method is actually used in multiple places and by having it as a separate method, we can centralize the error handling.